monero-outreach-docs/translations/nl/quickfacts/quickfacts.md

```diff
@@ -0,0 +1,134 @@
+# MONERO
```

Comment:
# MONERO

Monero is veilig, ontraceerbaar, digitaal privévaluta, beschikbaar voor iedereen.

# QUICK FACTS

Gebruikers kunnen in alle veiligheid spenderen, niemand kan hun balans zien of hun activiteit in kaart brengen.

[?] Revised 11/14/2018 - [Printer Friendly Version](http://www.monerooutreach.org/pubs/2018/QuickFacts/QuickFacts_PrinterFriendly.pdf)    
Created for the community by Monero Outreach

- [Full Color Version](http://www.monerooutreach.org/pubs/2018/QuickFacts/QuickFacts.pdf)

[getmonero.org](https://getmonero.org/)


# Kernprincipes

#### DECENTRALISATIE

Gezien Monero een _[open-bronproject](https://github.com/monero-project/monero)_ is geleid door een _[gedecentraliseerd team](https://getmonero.org/community/team/)_ van programmeurs en leden van de gemeenschap, is er geen censuur mogelijk. De meeste bijdrage komt van vrijwilligers van over de hele wereld.

#### BEVEILIGING

Monero is een gedecentraliseerde cryptovaluta omdat het beveiligd wordt door een wereldwijd netwerk van gebruikers. Transacties worden bevestigd middels gedistribueerde consensus en vervolgens onwijzigbaar opgenomen in de _[blokketen](https://www.mycryptopedia.com/what-is-blockchain-technology/)_.

#### SCHAALBAARHEID

De _[dynamische blokgrootte](https://www.mycryptopedia.com/block-size-explained/)_ van Monero varieert naargelang transactievolume om kosten, en transactietijd, laag te houden. Meer volume verhoogt de maximale blokgrootte, minder volume verlaagt het. _[Bulletproofs](https://web.stanford.edu/~buenz/pubs/bulletproofs.pdf)_ helpen de schaalbaarheid door significante reductie van transactiegrootte en -kosten. Zo blinkt Monero uit als cryptovaluta.

#### CENSUURBESTENDIG

Transacties op Monero's blokketen zijn niet verbonden aan welbepaalde gebruikers of hun dagelijkse identiteit. Daarom is censuur, en tevens _[kapitaalbeperking](https://en.wikipedia.org/wiki/Capital_control)_, onmogelijk.

#### PRIVACY

Monero gebruikt gesofisticeerde cryptografie als _[ring signatures](https://getmonero.org/resources/moneropedia/ringsignatures.html)_, _[ring confidential transactions](https://www.mycryptopedia.com/monero-ring-confidential-transactions-ringct/)_, and _[stealth addresses](https://www.mycryptopedia.com/everything-need-know-stealth-addresses/)_ om van alle transacties zowel de afkomst, bedrag, als bestemming te verduisteren.

#### FUNGIBILITEIT

Monero is _[fungibel](https://getmonero.org/resources/moneropedia/fungibility.html)_ omdat het standaard privé is. Dit betekent dat één Monero altijd gelijk zal zijn aan een ander. Verkopers of beurzen kunnen geen onderscheid maken tussen eenheden Monero op basis van de afkomst of geschiedenis van je munten.

#### GESCHIEDENIS VAN MONERO

Monero werd gelanceerd in April 2014. Het was een eerlijke, aangekondigde lancering van de **[CryptoNote](https://cryptonote.org/whitepaper.pdf)** code. Er was geen pre-mine of insta-mine, en geen deel van de blokbeloning gaat naar ontwikkeling. Zie de oorspronkelijke **[Bitcointalk thread](https://bitcointalk.org/index.php?topic=563821.0)**.

Sinds de lancering heeft Monero verschillende aanzienlijke verbeteringen doorgemaakt. Bijna allemaal betrof het vooruitgang in beveiliging of privacy, of gebruiksgemak. Monero blijft groeien met beveiliging en privacy als primair doel, en als secundair doel gebruiksgemak en efficiëntie.

### WAT BETEKENT MONERO

Het woord "Monero" komt van Esperanto. De bedenkers hebben deze taal gekozen omdat hij "gedecentraliseerd" is en symbool staat voor het breken van barrières tussen mensen wereldwijd. In Esperanto is "Monero" vrij opgebouwd uit drie elementen, elk één lettergreep: mon + er + o. Alle hebben ze betekenis.

**mon:** geld    
**er:** kleinste deel    
**o:** een ding / een zelfstandig naamwoord   

Dus "Monero" kan je analyseren als zijnde: "een zelfstandig naamwoord dat het kleinste deel van een geldmiddel beschrijft". Oftewel, een munt.

### BELANGRIJKE VERSCHILLEN

**Monero gebruikt de basiscode van Cryptonote:** Dit is een fundamenteel verschil met basiscode gebruikt door Bitcoin of Ethereum, en de vele andere cryptovaluta's afgeleid van beide. Het staat bekend om z'n opmerkelijke privacyverbeteringen.

**Privacy is verplicht; transparantie is optioneel:** Monero's grensverleggende cryptografie verduistert elke laag in een transactie: informatie over de zender, ontvanger, en de transactie zelf. Als een gebruiker transparantie verkiest, dan kan hij opteren een alleen-lezen wallet te delen die inputs onthult.

**Regelmatige netwerkupgrades:** De gemeenschap van Monero's programmeurs voert routine-upgrades uit aan het netwerk (hard forks) zodat all gebruikers optimale beveiliging, privacy en functionaliteit kunnen genieten. Door zich te schikken naar zowel opportuniteit als bedreiging, blijft het Moneronetwerk agiel en veilig. **[Hoe zit het met die hard forks die ik overal zie?](https://bitcoinmagazine.com/articles/monero-just-hard-forked-and-it-resulted-four-new-projects/)**

**Het traject van Monero's blokbeloning:** Beloningen zullen geleidelijk afnemen tot aan de staartemissie einde mei 2022, tot een vaste beloning van 0.6 XMR per blok. Staartemissie biedt voortdurende en onbegrensde mijnstimulans. Bovendien geven staartemissies Monero een ingebouwde, stabiele en voorspelbare inflatie. Dit is essentieel voor reëel, degelijk geld.

**Monero Research Lab:** Monero wil niet alleen een fungibele valuta zijn, maar ook doorgaan met onderzoek voeren naar financiële privacy.
Hiervoor hebben researchers in wiskunde en numerieke natuurkunde **[white papers](https://lab.getmonero.org/)** gepubliceerd, en hebben ze vele overige **[onderzoeksdoelen](https://www.getmonero.org/design-goals/)**.

**Mijnen is toegankelijk:** Iedereen met een verbonden apparaat of webbrowser kan deelnemen.


### PRAKTISCHE GEVOLGEN & TOEPASSINGEN

Omdat Monero veilig, goedkoop en grenzeloos is, kunnen mensen eenvoudig geld verzenden ondanks corrupte, falende overheden of banken. Dit zet mensen economisch op de kaart in **[onderdrukte landen](https://www.reddit.com/r/Monero/comments/6wczty/how_monero_changed_my_life/)** of in depressieve economieën.

Financiële geheimhouding beschermt consumenten en bedrijven tegen prijsmanipulatie, uitbuiting binnen de productieketen, economische discriminatie, enz... **[Monero is de enige cryptovaluta](https://www.reddit.com/r/Monero/comments/8k8pk9/monero_the_worlds_bestkept_secret/)** met de kenmerken van volledig fungibel, gedecentraliseerd, elektronisch geld.

### TECHNISCHE GRONDBEGINSELEN

_(Peildatum: 14/11/2018)_

 + **Aantale actieve nodes:** 1,756 (bron: https://monerohash.com/nodes-distribution.html)
+ **Netwerk hash rate:** 435.17 MegaHashes/s
+ **Gemiddeld aantal transacties/uur:** 178 (30-daags gemiddelde)
+ **Aantal CPU cores die het netwerk beveiligen:** 14,499,178
+ **Aantal Monero in omloop:** 16,570,126 XMR (schatting)
+ **Marktkapitalisatie:** $1,503,928,296 USD (~0.77% van de totale marktkapitalisatie in cryptovaluta)
+ **Huidige hoogte van de blokbeloningen:** 3.5984 XMR
+ **Gemiddelde blokinterval:** 2 Minutes

_Blokbeloning zal geleidelijk afnemen tot eind mei 2022, wanneer er  18.132 miljoen XMR circuleert, en de beloning vast komt te staan op 0.6 XMR per blok._

_Met een staartemissie van 0.6 XMR/blok zal er in 2040 evenveel Monero als Bitcoin zijn (ongeveer 21 miljoen)._

### TOEKOMSTIGE FEATURES

Hoewel Monero beschikbaar is en al gebruikt wordt over de hele wereld, hebben de programmeurs interessante plannen om privacy, veiligheid, en functionaliteit van Monero (en cryptovaluta in het algemeen) te blijven verbeteren. Deze features komen er binnenkort aan:

**Bulletproofs:** Deze [technologie](https://getmonero.org/es/2017/12/07/Monero-Compatible-Bulletproofs.html) slaat de blokketen efficiënter op en biedt snellere verifiëring van transacties.

**Kovri:** Voor privacy is dit een belangrijke upgrade. Kovri voegt een vierde laag privacy toe aan Monero-transacties. Hiertoe gebruikt het **[garlic-encryption](https://getmonero.org/resources/moneropedia/garlic-encryption.html)** (i.e. TOR's grote broer) door het IP-adres van zenders en ontvangers te verbergen. Elke laag van een Monero-transactie zal privé en veilg zijn: van zender, ontvanger, bedrag, tot het überhaupt gebruik van het Moneronetwerk (dankzij Kovri).

**Hardware Wallets:** Trezor Model T hardware wallet **[ondersteunt gebruik van Monero](https://trezor.io/coins/#XMR)**. **[Ledger hardware wallet](https://support.ledgerwallet.com/hc/en-us/articles/360006352934-Monero-XMR-Advanced)** is al geïntegreerd via de officiële Monero GUI wallet. De gemeenschap sponsert een team dat **[Kastelo, een open-bron hardware wallet](http://kastelo.org/)** bouwt, volgens de Monero-ethos. Beide opties worden volledig ondersteund.

**Kasisto:** Dit **[computergestuurd kasregister](https://github.com/amiuhle/kasisto)** is ontworpen voor verkoopsgemak van handelaars en 
bedrijven.


## EXTRA BRONMATERIAAL

+ **_[getmonero.org (Officiële website)](https://getmonero.org/)_**
+ **_[monero.how](https://www.monero.how/)_**
+ **_[reddit.com/r/monero](https://www.reddit.com/r/Monero/)_**
+ **_[Gids voor Monero (post)](https://www.reddit.com/r/CryptoCurrency/comments/7ra409/your_guide_to_monero_and_why_it_has_great/)_**
+ **_[diepgaande technische introductie tot Monero](https://steemit.com/monero/@sgp/7yjqso-a-monero-introduction-for-beginners)_**
+ **_[Zero to Monero](https://www.getmonero.org/library/Zero-to-Monero-1-0-0.pdf)_**

 + **_[Vermijd deze oplichters](https://www.reddit.com/r/Monero/wiki/avoid)_**
+ **_[Monero FAQ](https://ww.getmonero.org/get-started/faq/)_**
+ **_[Monero SWOT analyse]()_**
+ **_[de Monero gemeenschap](https://getmonero.org/community/hangouts/)_**
+ **_[Mastering Monero](https://masteringmonero.com/)_**

 **Monero Quick Facts - Gereviseerd op 14/11/2018**    
**Geschreven voor de Monero-gemeenschap door Monero Outreach.**

 **Monero Outreach heeft als doel om adoptie en gebruik van Monero te stimuleren door middel van educatie en public relations. Jouw donaties maken dit mogelijk.**

 **47oKHkoaQdBdFpTJNKaetUS6UsCGHVbJbGxPGaaHFQPqXSCLbqXYsBo6x7abwtfdXTeiBhtZLnYF5bRRAhYsUVb5Sd1aqiD**